crates/store/re_protos/proto/rerun/v1alpha1/cloud.proto

```diff
@@ -346,6 +358,24 @@ message CreateIndexResponse {
   reserved "data";
 }
 
+message ListIndexesRequest {}
+
+message ListIndexesResponse {
+  repeated IndexConfig indexes = 1;
+}
+
+message DeleteIndexesRequest {
+  // Which column to delete the indexes for.
+  IndexColumn column = 1;
+}
+
+message DeleteIndexesResponse {
+  // Which indexes were actually deleted.
+  //
+  // Can be empty if no matching indexes were found.
+  repeated IndexConfig indexes = 1;
```

Comment:
Note: this will never have more than 1 entry as of today, as we only allow 1 index per column descriptor. We expect this limitation to be lifted in the future though, so we're future-proofing for that.